core/mobile/src/backend/MobileHost.ts

```diff
@@ -153,10 +165,33 @@ export class MobileHost {
 
   /** Start the backend of a mobile app. */
   public static async startup(opt?: MobileHostOpts): Promise<void> {
+    const authorizationClient = new MobileAuthorizationBackend();
```

Comment:
I don't think you'll want to create and set this prior to the device being setup. I don't think you immediately try to use it but could lead to weird issues.